guides/hack/00-overview/03-async.md

```diff
@@ -1,85 +1,15 @@
 ### Async Programming in Hack

```

Comment:
Yeah - the guide controllers were always deriving a page title from the Guide's filename (in this case for example "Overview: Async") and I think we need to ditch that autogenerated page title, that way having the `<h1>` following them won't be bad for SEO/accessibility purposes.